src/site/content/en/fast/content-delivery-networks/index.md

```diff
@@ -268,7 +268,7 @@ Although switching your CDN instance to HTTP/2 is largely a matter of flipping a
     HTTP/3 uses TLS 1.3 and therefore shares its performance benefits: establishing a new connection only requires a single round-trip and resuming an existing connection does not require any roundtrips.
 
 <figure class="w-figure">
-  <img src="./cdn5.png" alt="Comparison of connection resumption between TLS 1.2 and TLS 1.3" class="w-screenshot">
+  <img src="./cdn5.png" alt="Comparison of connection resumption between TLS 1.2, TLS 1.3, TLS 1.3 0-RTT, and HTTP/3" class="w-screenshot">
```

Comment:
I can't seem to comment on the new image itself via GitHub UI, so I'll try it here.

Main comment: the asterisk next to 1 RTT should be the double asterisk to match the legend on the bottom

Other than that, I think this approach is technically correct and better than the previous image. However, I wonder if the bottom right 0 RTT will feel strange to some readers, as it seems no network traffic is being exchanged. In similar images, I personally have taken to including the first HTTP exchange in all sequences as well to (hopefully) prevent this confusion. I'm not sure if that's needed here, but maybe something to consider.